logistics/venue.md

```diff
@@ -0,0 +1,99 @@
+# Venue Logistics
+
+RustBridge is an interactive event and your space should allow for that. That
+means you should have space for people to walk around and people to comfortably
+sit in groups.
+
+Learning works better when having a little space and it's not too noisy.
+
+Don't stress out about venue details _too_ much, people are great at working
+around flaws, as long as you and the attendees are aware. It's the learning 
+experience that counts.
+
+## A venue is a relationship
+
+Be aware that venues are run by people and you want to be on good terms with
+them. If you want to run multiple events and you are happy with a venue,
+you totally want to leave a lasting impression. This will make searching
+for the next time easier.
+
+If you want to run regular workshops, make sure you build a couple of contacts.
+
+## Qualities of a good venue
+
+RustBridge is a flexible program, so you are also flexible in a venue.
+
+A venue for RustBridge should fulfill the following requirements:
+* It's free
```

Comment:
Presumably "free for the attendees to enter"?

It's possible to have funding for such an event at a paid venue (e.g. when such events are run at a conference) where it is free to attend the event itself

Reply:
Sure, but from the view of RustBridge, that's free.

Funding also means bookkeeping and someone to send the invoice to. Many smaller groups don't have that.

I changed it to "you don't have to pay for it".